.gitignore

```diff
@@ -0,0 +1,7 @@
+.Rproj.user
+.Rhistory
+.RData
+.Ruserdata
+1_fetch/in/*
+!1_fetch/in/saline_lakes.csv
+!1_fetch/in/states_shp
```

Comment:
Ignore the shapefiles so we are not committing them to github

Reply:
are you suggesting we do not push the states shp? I was thinking it might be easiest to include the states shapefile since it's not so heavy and because it comes from a source that cannot be downloaded with sbtools.

Reply:
We try to avoid committing almost all data files for our viz projects. Alternative options to keep everything within the pipeline would be:
1. use the download link from sb to access https://prd-tnm.s3.amazonaws.com/StagedProducts/Small-scale/data/Boundaries/statesp010g.shp_nt00938.tar.gz 
2.  access the census boundaries through `tigris` https://github.com/walkerke/tigris

Reply:
addressed by pulling directly from link provided above. Addressed in 
https://github.com/USGS-VIZLAB/saline-lakes/pull/2/commits/69df1e6dd780c900154639336d098529568b73dd